### ISSUE TITLE
Question about using StandardScaler().fit_transform() on  Prediction phase

### ISSUE THREAD

**Issue by 2seunghoon** [cure-lab/LTSF-Linear#21]
Hello

In dataloader.py -> Dataset_Pred
In general, the distribution of test dataset is unknown, so StandardScaler().fit() is not given, but StandardScaler().fit() is given in the Dataset_Pred. May I know why?

**Comment by mixiancmx** [comment]
I think it is like an instance normalization for the input. Models are trained with a standard normalized dataset. Therefore, when we use them on realistic input, we need to normalize the input too.

**Comment by mixiancmx** [comment]
The distribution of the test dataset is unknown, but we at least have an input for the model. This StandardScaler() is on the input.
